STM32F405RGTx/Core/Src/main.c

```diff
@@ -718,10 +720,37 @@ void Error_Handler(void)
 {
   /* USER CODE BEGIN Error_Handler_Debug */
   /* User can add his own implementation to report the HAL error return state */
+  
+  /* Disable all interrupt requests */
   __disable_irq();
+
+  /* Slave device does not require disabling. SLEEP state activated automatically */
+  /* In STADNDY, REFUP, MEASURE states, SLEEP state reached after t_sleep time */
+  /* Other states are not reached, which can be ignored */
+
+  /* Report a fault on CAN */
+  __report_CAN();
+
+  /* Report a fault on UART */
+  printf("Error handling initiated.");
+
+  /* Disengage contactor */
+  TURN_OFF_CONTACTOR_PIN();

```

Comment:
When you add the extra contactor pin to the bsp, make sure to add a turn off command to the exception handler